src/jpeg/entropy.rs

```diff
@@ -1,24 +1,26 @@
+#![feature(const_fn)]
```

Comment:
We're not switching to nightly :confused:

Reply:
Fair enough, I've reverted the changes.